### PR TITLE
[REF] account_refund_early_payment: The refunds are created in base o… task#29946

### PR DESCRIPTION
**Pull request by keylor2906**
…f the amount total of the wizard instead of the percentage value, the percentage now is only used to get the amount total on change.

### CODE REVIEW

**Comment by keylor2906** [comment]
@dsabrinarg could you review this, please?

Dev instance is http://dev.lodigroup.com:30011/

dummy mr https://git.vauxoo.com/vauxoo/quebec/merge_requests/20

**Comment by sabrinagnu** [comment]
@yaniaular may you please give me your approval here?. Regards!.

**Comment by sabrinagnu** [comment]
@keylor2906 functional test 👍

**Review by yaniaular — APPROVED** [review]

File: account_refund_early_payment/wizard/account_invoice_refund.py [hunk 1]
```diff
@@ -48,14 +48,19 @@ def compute_refund(self, mode):
         for inv, refund in zip_longest(invoices, refunds, fillvalue=None):
             if not inv or not refund:
                 break
+            percentage = inv.amount_total / total
+            taxes = self.product_id.taxes_id
+            tax_perc = 0
+            for tax in taxes:
+                tax_perc += tax.amount
```
Comment (josemoralesp) [inline account_refund_early_payment/wizard/account_invoice_refund.py]: @keylor2906 

```python
tax_perc = sum(taxes.mapped('amount'))
```

**Pull request merged**